rest/route/service.go

```diff
@@ -203,6 +203,7 @@ func AttachHandler(app *gimlet.APIApp, opts HandlerOpts) {
 	app.AddRoute("/projects/{project_id}/task_reliability").Version(2).Get().Wrap(requireUser).RouteHandler(makeGetProjectTaskReliability(opts.URL))
 	app.AddRoute("/projects/{project_id}/task_stats").Version(2).Get().Wrap(requireUser, viewTasks).RouteHandler(makeGetProjectTaskStats(opts.URL))
 	app.AddRoute("/projects/{project_id}/versions").Version(2).Get().Wrap(requireUser, viewTasks).RouteHandler(makeGetProjectVersionsHandler(opts.URL))
+	app.AddRoute("/projects/{project_id}/versions").Version(2).Post().Wrap(requireUser, submitPatches).RouteHandler(makeSetPriorityProjectVersionsHandler(opts.URL))
```

Comment:
Discussed IRL that we may want to make this a project admin permission since it could be abused.